### PR TITLE
nix: fix "common.sh: line 94: unshare: command not found"

### PR DESCRIPTION
**Pull request by vobloeb**
See
https://github.com/NixOS/nixpkgs/pull/111871#issuecomment-773351871
https://github.com/NixOS/nixpkgs/pull/95742#issuecomment-675472428
https://github.com/NixOS/nixpkgs/issues/56106#issuecomment-492375471

### CODE REVIEW

File: pkgs/tools/package-management/nix/default.nix [hunk 1]
```diff
@@ -10,7 +10,7 @@ let
 
 common =
   { lib, stdenv, perl, curl, bzip2, sqlite, openssl ? null, xz
-  , bash, coreutils, gzip, gnutar
+  , bash, coreutils, utillinux, gzip, gnutar
```
Comment (cole-h) [inline pkgs/tools/package-management/nix/default.nix:13]:
```suggestion
  , bash, coreutils, util-linux, gzip, gnutar
```

File: pkgs/tools/package-management/nix/default.nix [hunk 2]
```diff
@@ -40,7 +40,7 @@ common =
       outputs = [ "out" "dev" "man" "doc" ];
 
       nativeBuildInputs =
-        [ pkg-config ]
+        [ pkg-config utillinux ]
```
Comment (cole-h) [inline pkgs/tools/package-management/nix/default.nix:43]:
```suggestion
        [ pkg-config util-linux ]
```

**Review by cole-h — CHANGES_REQUESTED** [review]
`utillinux` is an alias to `util-linux`.

**Comment by SuperSandro2000** [comment]
Please squash the commits together.

**Pull request closed without merging**